### DISCUSSION TITLE
Registration in fb2k component list

### DISCUSSION THREAD

**Discussion by bassstorm** [ghDaYuYu/foo_discogs#104] [General]
Thanks for great work on continuation of foo_discogs component development and maintenance! Works like a charm with fb2k x64 here :-)

To give it a wider attention, it's worth to put it on the actual fb2k [component list](https://www.foobar2000.org/components), which has only outdated original. I guess I'm not the first one who's asking this question, but was unable to find anything in discussions, so please bear with me.

Cheers!

**Comment by ghuDaYuYu** [comment]
Glad you enjoy using Discogger.
It has been requested before... but just once or so, and at the time I thought of it as a Himalayan blunder.

Not so sure now. The issues section is quiet, as well as the support forum.
You are probably right, maybe it is time to make a referral to the inclusion.
I will report here any news as soon as possible.
Thanks for your feedback...

Regards!

**Comment by bassstorm** [comment]
Great to hear :)
I've just submitted #105 which I came across personally.

**Comment by ghuDaYuYu** [comment]
About the registration, there is no news... 
But no news is also good news  : )
Too much attention could easily slow down development,
and it is not the amount but the quality of attention that matters.

Best regards!
